portality/tasks/journal_in_out_doaj.py

```diff
@@ -61,6 +61,9 @@ def run(self):
             j.set_in_doaj(in_doaj)
             j.save()
             j.propagate_in_doaj_status_to_articles()  # will save each article, could take a while
+            if not in_doaj:
+                job.add_audit_message("Rejecting all associated update requests")
+                j.reject_update_requests()
```

Comment:
I think this needs to change.  We want this to be in the application service, where related methods are, like `reject_journal`.

So, here we would have a call to 

```
svc = DOAJ.applicationService()
svc.reject_update_requests()
```

and in `portality.bll.service.application` then we want the contents of `Journal.reject_update_requests` to be reflected as a BLL method rather than a model method.

Reply:
Also, it would be nice if this could report on the update requests rejected, so have that `reject_update_requests` return some status saying whether it has rejected them or not.  This could be a true or false, or a count (there should be only one) or an object with some status informaiton.  Not sure which is best, probably the count.  Then this could be logged in the audit message